webapp/src/ts/services/rules-engine.service.ts

```diff
@@ -55,20 +55,21 @@ export class RulesEngineService implements OnDestroy {
   private debounceActive: DebounceActive = {};
 
   constructor(
-    private translateService: TranslateService,
-    private authService: AuthService,
-    private sessionService: SessionService,
-    private settingsService: SettingsService,
-    private telemetryService: TelemetryService,
-    private uhcSettingsService: UHCSettingsService,
-    private userContactService: UserContactService,
-    private userSettingsService: UserSettingsService,
-    private parseProvider: ParseProvider,
-    private changesService: ChangesService,
-    private contactTypesService: ContactTypesService,
-    private translateFromService: TranslateFromService,
-    private rulesEngineCoreFactoryService: RulesEngineCoreFactoryService,
-    private calendarIntervalService: CalendarIntervalService
+    private translateService:TranslateService,
```

Comment:
This is just a comment not a change request:

I know the different preference here, but I find it hard to read. Specially something long like: rulesEngineCoreFactoryService:RulesEngineCoreFactoryService
😅 I wish Typescript people came up with something diff, I don't know like: classA::classB  that extra "space" between helps me to read

Reply:
For me it's the opposite. Having a space makes me think of object properties assignment. 
I feel like: 
```
{ property:type : value }
```
makes much more sense than:
```
{ property: type : value }
```